### PR TITLE
style lists from stimulus in nursing-internal

### PR DESCRIPTION
**Pull request by jbwilson8**
Issue: [5359](https://github.com/openstax/cnx-recipes/issues/5359)

Screenshot 2023-12-13 at 4 25 32 PM

### CODE REVIEW

File: styles/designs/carnival/parts/lists/_lists-components.scss [hunk 1]
```diff
@@ -10,6 +10,14 @@ $UnorderedList--FirstLevel: (
...
 );
 
+$UnorderedList--FirstLevel_FromStimulus: (
+    _name: "FirstLevelUnorderedListFromStimulus",
+    _subselector: ' > ul > li',
+    _properties:(
+        margin-left: h-spacing(3),
+    )
+);
+
```
Comment (wilkus27) [inline styles/designs/carnival/parts/lists/_lists-components.scss:20]: We don't need new component. It has the same properties as regular `$UnorderedList--FirstLevel`. We just have to put it under a proper wrapper in a shape.

File: styles/designs/carnival/parts/lists/_lists-from-modules-shapes.scss [hunk 2]
```diff
@@ -165,3 +165,16 @@
...
+
+@include create_shape('carnival:::ListsFromStimulus',(
+    _components: (
+        map-merge($ListContainer,(
+          _components: (
+            $UnorderedList--FirstLevel_FromStimulus,
+          )),
+        )
+    )
+));
```
Comment (wilkus27) [inline styles/designs/carnival/parts/lists/_lists-from-modules-shapes.scss:180]: Could you move this to this file https://github.com/openstax/ce-styles/blob/main/styles/designs/carnival/parts/lists/_lists-from-elements-shapes.scss ? There is already shape for exercises, but for now we can't reuse it for stimulus, so lets keep a new shape. More info is in a corn: https://github.com/openstax/ce-styles/blob/31c8eddd8b577cc7887dc8ef34761626df70afb0/styles/designs/corn/parts/lists/_lists-from-elements-shapes.scss#L47

Comment (wilkus27) [reply styles/designs/carnival/parts/lists/_lists-from-modules-shapes.scss:180]: And please remember to add word `Shape` to the name of a shape (`ListsFromStimulusShape`)

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: styles/designs/carnival/parts/lists/_lists-components.scss (hunk 1 above)
Comment (jbwilson8) [reply styles/designs/carnival/parts/lists/_lists-components.scss]: Thanks, I've removed the redundant component :-)

File: styles/designs/carnival/parts/lists/_lists-from-modules-shapes.scss (hunk 2 above)
Comment (jbwilson8) [reply styles/designs/carnival/parts/lists/_lists-from-modules-shapes.scss]: Also, I updated the shape name, thanks for your feedback :-)

**Review by wilkus27 — APPROVED** [review]
Looks good :wink: :+1:

**Pull request merged**